SilverSolver/assignment1/Q1.py

```diff
@@ -0,0 +1,49 @@
+def are_words_anagram(s1, s2, case_sensitive=False, return_dicts=False):
+    alphabet = "abcdefghijklmnopqrstuvwxyz"
+    if not case_sensitive:
+        s1 = s1.lower()
+        s2 = s2.lower()
+    else:
+        alphabet += alphabet.upper()
+    alphabet += "0123456789"
+    s1 = s1.strip()         # Ignoring spaces
+    s2 = s2.strip()
+
+    char_counter_1 = dict.fromkeys(list(alphabet), 0)
```

Comment:
I like this approach with the dict counting the occurences. There is such a a structure already in the standard library: it returns the dict of occurences when you give it an iterable.
collections.Counter

Can you use it?

https://docs.python.org/3/library/collections.htm